packages/backend/src/config/parseConfig.ts

```diff
@@ -1635,7 +1635,7 @@ export const parseConfig = (): LightdashConfig => {
             enabled: process.env.MCP_ENABLED === 'true',
         },
         customRoles: {
-            enabled: process.env.CUSTOM_ROLES_ENABLED === 'true',
+            enabled: true,
```

Comment:
Do we want to enable custom roles for everyone now?

Reply:
rollback